cmake/FindLibLUV.cmake

```diff
@@ -20,7 +20,8 @@ if(LIBLUV_USE_STATIC)
     "${CMAKE_STATIC_LIBRARY_PREFIX}luv${CMAKE_STATIC_LIBRARY_SUFFIX}")
 endif()
 
-list(APPEND LIBLUV_NAMES luv)
+# Explicitly look for luv.so. #10407
+list(APPEND LIBLUV_NAMES luv luv${CMAKE_SHARED_LIBRARY_SUFFIX})
```

Comment:
I think this might make sense in general, but does not help by itself usually (at least for nixpkgs).
A better approach is passing linker flags directly: https://github.com/NixOS/nixpkgs/issues/64400#issuecomment-517930779

Reply:
Should at least mention that in https://github.com/neovim/neovim/wiki/Building-Neovim , or maybe in the main README.md